MAUI/DataGrid/sorting.md

```diff
@@ -245,6 +245,34 @@ public class CustomSortComparer : IComparer<object>, ISortDirection
 {% endhighlight %}
 {% endtabs %}
 
+## Change sort icon color
+
+The default sort icon color can be customized by setting the `DataGridStyle.SortIconColor` property.
+{% tabs %}
+
+{% highlight xaml %}
+<syncfusion:SfDataGrid x:Name="sfDataGrid"
+                        ItemsSource="{Binding OrderInfoCollection}" >
+        <syncfusion:SfDataGrid.DefaultStyle>
+                <syncfusion:DataGridStyle SortIconColor="DodgerBlue" />
+        </syncfusion:SfDataGrid.DefaultStyle>
+            <syncfusion:SfDataGrid.Columns>
+                    <syncfusion:DataGridTextColumn MappingName="OrderID"
+                              AllowSorting="True"
+                              HeaderText="Order ID" />
+                    <syncfusion:DataGridTextColumn MappingName="CustomerID"
+                              HeaderText="Customer ID" />
+                    <syncfusion:DataGridTextColumn MappingName="ShipCountry "
+                              HeaderText="Ship Country " />
+            </syncfusion:SfDataGrid.Columns>
+</syncfusion:SfDataGrid>
+{% endhighlight %}
+{% endtabs %}
+
+
+<img alt="DataGrid with sort template selector" src="Images\sorting\maui-datagrid-sort-icon-color.png" width="404">

```

Comment:
Alt text should be "Sort icon customization in MAUI DataGrid"

Reply:
changed